pysys/utils/allocport.py

```diff
@@ -80,10 +85,20 @@ def initializePortPool():
 
 	global tcpServerPortPool
 
-	ephemeral_low, ephemeral_high = getEphemeralTCPPortRange()
-
-	# Allocate server ports from all non-privileged, non-ephemeral ports
-	tcpServerPortPool = list(range(1024, ephemeral_low)) + list(range(ephemeral_high,65536))
+	portsfile = os.getenv('PYSYS_PORTS_FILE',None)
```

Comment:
It would be cool if this file could contain ranges, as I would expect a common situation to be that I want to use all the ports in a large range except the ones I know are used for something else.  That is, somehow list the smaller set of exclusions not all the inclusions.  Definitely a feature request but I thought I should mention it since I thought of it.

Reply:
yes maybe, though as you say that's one for later. And easy to write a script to generate a file that does that - so at least the basic building block is present in PySYs now